client/signup/jetpack-connect/plan-features-tab/index.jsx

```diff
@@ -0,0 +1,378 @@
+
+const jetpackPlansPersonalTab = [ PLAN_JETPACK_FREE, PLAN_JETPACK_PERSONAL ];
+const jetpackPlansPremiumTab = [ PLAN_JETPACK_PREMIUM, PLAN_JETPACK_BUSINESS ];
+const jetpackMonthlyPlansPersonalTab = [ PLAN_JETPACK_FREE, PLAN_JETPACK_PERSONAL_MONTHLY ];
+const jetpackMonthlyPlansPremiumTab = [ PLAN_JETPACK_PREMIUM_MONTHLY, PLAN_JETPACK_BUSINESS_MONTHLY ];
+
+class PlansFeaturesMain extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = { currentTab: 'personal' };
+		this.changePlanGroup = this.changePlanGroup.bind( this );
+	}
+
+	changePlanGroup() {
+		this.state.currentTab =
+			this.setState( {
+				currentTab: this.state.currentTab === 'personal'
+					? 'premium'
+					: 'personal'
+			}
+		);
+	}
+
+	renderPlanTypeSelector() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<SegmentedControl primary className="plan-features-tab__selector">
+					<SegmentedControlItem
+						selected={ this.state.currentTab === 'personal' }
+						onClick={ this.changePlanGroup }>
+							{ translate( 'Personal' ) }
+					</SegmentedControlItem>
+					<SegmentedControlItem
+						selected={ this.state.currentTab === 'premium' }
+						onClick={ this.changePlanGroup }>
+							{ translate( 'Business' ) }
+					</SegmentedControlItem>
+				</SegmentedControl>
+			</div>
+		);
+	}
+
+	isJetpackSite( site ) {
+		return site.jetpack;
+	}
+
+	getCurrentViewablePlans() {
+		let plans;
+		if ( this.props.intervalType === 'monthly' ) {
+			plans = this.state.currentTab === 'personal'
+				? jetpackMonthlyPlansPersonalTab
+				: jetpackMonthlyPlansPremiumTab;
+		}
+		plans = this.state.currentTab === 'personal'
+			? jetpackPlansPersonalTab
+			: jetpackPlansPremiumTab;
+
+		if ( this.props.hideFreePlan && this.state.currentTab === 'personal' ) {
+			plans = Object.assign( [], plans );
+			plans.shift();
+		}
+
+		return plans;
+	}
+
+	getPlanFeatures() {
+		const {
+			site,
+			intervalType,
+			onUpgradeClick,
+			isInSignup,
+			isLandingPage,
+			basePlansPath,
+			selectedFeature
+		} = this.props;
+
+		if ( this.isJetpackSite( site ) && intervalType === 'monthly' ) {
+			return (
+				<div className="plans-features-main__group">
+					<PlanFeatures
+						plans={ this.getCurrentViewablePlans() }
+						selectedFeature={ selectedFeature }
+						onUpgradeClick={ onUpgradeClick }
+						isInSignup={ isInSignup }
+						isLandingPage={ isLandingPage }
+						basePlansPath={ basePlansPath }
+						intervalType={ intervalType }
+						site={ site }
+					/>
+				</div>
+			);
+		}
+
+		return (
+			<div className="plans-features-main__group">
+				<PlanFeatures
+					plans={ this.getCurrentViewablePlans() }
+					selectedFeature={ selectedFeature }
+					onUpgradeClick={ onUpgradeClick }
+					isInSignup={ isInSignup }
+					isLandingPage={ isLandingPage }
+					basePlansPath={ basePlansPath }
+					intervalType={ intervalType }
+					site={ site }
+				/>
+			</div>
+		);
+	}
+
+	getJetpackFAQ() {
```

Comment:
`getJetpackFAQ` and `getFaq` could be separate stateless components in their own files and then we could just import and use `<JetpackFaqs />` and `<SiteFaqs site={ site } />`